## extracted.matrix

The file contains 294 profiles of elementary functional loops (EFLs).

The format is simle and starts with a header followed by positional frequencies for each position and amino acid residue in the profile.

See example below, where ID is Profile ID, K is the number of sequences in the alignment used to obtain the profile and L is the length of the profile (the number of positions).

Each of the following lines corresponds to a position in the profile and is numbered sequentially starting from 0.

    PROFILE 4
    BEGIN
    ORIGIN binding_Ca_Zn; PDOC00018; PS00018; 1
    MATRIX ID=0 K=353 L=30
    30        A        C        D        E        F        G        H        I        K        L        M        N        P        Q        R        S        T        V        W        Y
     0 0.105714 0.028571 0.000000 0.002857 0.065714 0.008571 0.002857 0.157143 0.005714 0.311429 0.025714 0.000000 0.005714 0.014286 0.002857 0.048571 0.022857 0.140000 0.028571 0.022857
     1 0.028571 0.005714 0.108571 0.151429 0.014286 0.017143 0.017143 0.042857 0.160000 0.037143 0.025714 0.051429 0.014286 0.077143 0.120000 0.040000 0.040000 0.022857 0.002857 0.022857
     2 0.074074 0.005698 0.096866 0.136752 0.037037 0.025641 0.025641 0.028490 0.111111 0.037037 0.008547 0.039886 0.005698 0.079772 0.102564 0.068376 0.042735 0.031339 0.019943 0.022792
     3 0.128205 0.019943 0.005698 0.017094 0.082621 0.008547 0.005698 0.168091 0.002849 0.145299 0.113960 0.002849 0.000000 0.017094 0.014245 0.025641 0.048433 0.150997 0.025641 0.017094
     4 0.025568 0.000000 0.000000 0.000000 0.676136 0.000000 0.000000 0.082386 0.000000 0.076705 0.051136 0.000000 0.000000 0.000000 0.005682 0.000000 0.002841 0.025568 0.034091 0.019886
     5 0.053977 0.005682 0.122159 0.090909 0.011364 0.017045 0.025568 0.014205 0.184659 0.019886 0.011364 0.076705 0.005682 0.093750 0.119318 0.071023 0.042614 0.025568 0.005682 0.002841
     6 0.082386 0.008523 0.039773 0.136364 0.011364 0.017045 0.028409 0.062500 0.082386 0.178977 0.056818 0.022727 0.000000 0.042614 0.076705 0.048295 0.025568 0.065341 0.000000 0.014205
     7 0.101983 0.039660 0.000000 0.000000 0.223796 0.008499 0.016997 0.121813 0.000000 0.121813 0.065156 0.008499 0.000000 0.002833 0.002833 0.045326 0.008499 0.087819 0.008499 0.135977
     8 0.000000 0.000000 1.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000
     9 0.079320 0.000000 0.019830 0.045326 0.031161 0.016997 0.011331 0.053824 0.229462 0.093484 0.011331 0.011331 0.022663 0.036827 0.062323 0.059490 0.121813 0.073654 0.002833 0.016997
    10 0.000000 0.000000 0.665722 0.000000 0.000000 0.000000 0.000000 0.000000 0.011331 0.000000 0.000000 0.314448 0.000000 0.000000 0.000000 0.008499 0.000000 0.000000 0.000000 0.000000
    11 0.005666 0.008499 0.011331 0.036827 0.002833 0.371105 0.048159 0.000000 0.189802 0.005666 0.005666 0.101983 0.000000 0.059490 0.096317 0.053824 0.002833 0.000000 0.000000 0.000000
    12 0.000000 0.000000 0.626062 0.008499 0.000000 0.000000 0.002833 0.000000 0.019830 0.000000 0.000000 0.178470 0.000000 0.000000 0.000000 0.164306 0.000000 0.000000 0.000000 0.000000
    13 0.000000 0.000000 0.000000 0.000000 0.000000 0.923513 0.000000 0.000000 0.008499 0.000000 0.000000 0.042493 0.000000 0.000000 0.025496 0.000000 0.000000 0.000000 0.000000 0.000000
    14 0.053824 0.016997 0.005666 0.053824 0.079320 0.011331 0.031161 0.036827 0.161473 0.062323 0.014164 0.025496 0.000000 0.050992 0.065156 0.059490 0.110482 0.062323 0.000000 0.099150
    15 0.000000 0.000000 0.000000 0.000000 0.005682 0.000000 0.000000 0.502841 0.000000 0.289773 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.201705 0.000000 0.000000
    16 0.002849 0.000000 0.245014 0.054131 0.000000 0.037037 0.005698 0.000000 0.000000 0.000000 0.000000 0.094017 0.005698 0.014245 0.000000 0.324786 0.216524 0.000000 0.000000 0.000000
    17 0.057143 0.011429 0.028571 0.037143 0.194286 0.014286 0.014286 0.074286 0.100000 0.120000 0.017143 0.011429 0.031429 0.020000 0.094286 0.028571 0.025714 0.040000 0.017143 0.062857
    18 0.051576 0.002865 0.157593 0.252149 0.011461 0.017192 0.017192 0.008596 0.108883 0.025788 0.008596 0.088825 0.014327 0.063037 0.042980 0.063037 0.025788 0.017192 0.000000 0.022923
    19 0.000000 0.000000 0.106017 0.893983 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000
    20 0.017192 0.000000 0.000000 0.000000 0.401146 0.000000 0.002865 0.091691 0.000000 0.260745 0.060172 0.000000 0.000000 0.000000 0.000000 0.002865 0.000000 0.063037 0.037249 0.063037
    21 0.068966 0.031609 0.031609 0.066092 0.011494 0.037356 0.020115 0.080460 0.120690 0.129310 0.014368 0.040230 0.002874 0.074713 0.077586 0.054598 0.037356 0.083333 0.002874 0.014368
    22 0.129683 0.020173 0.072046 0.095101 0.014409 0.037464 0.034582 0.034582 0.074928 0.040346 0.020173 0.063401 0.000000 0.086455 0.054755 0.066282 0.083573 0.043228 0.002882 0.025937
    23 0.176301 0.037572 0.008671 0.014451 0.101156 0.083815 0.014451 0.078035 0.011561 0.138728 0.075145 0.000000 0.000000 0.008671 0.008671 0.028902 0.028902 0.095376 0.026012 0.063584
    24 0.052174 0.026087 0.005797 0.002899 0.130435 0.023188 0.008696 0.113043 0.014493 0.266667 0.156522 0.011594 0.002899 0.023188 0.002899 0.005797 0.017391 0.095652 0.011594 0.028986
    25 0.060870 0.000000 0.026087 0.055072 0.017391 0.040580 0.049275 0.043478 0.107246 0.066667 0.037681 0.031884 0.005797 0.072464 0.127536 0.086957 0.101449 0.043478 0.002899 0.023188
    26 0.089595 0.005780 0.060694 0.069364 0.005780 0.037572 0.014451 0.034682 0.153179 0.086705 0.034682 0.031792 0.026012 0.031792 0.075145 0.130058 0.060694 0.037572 0.008671 0.005780
    27 0.028902 0.031792 0.034682 0.028902 0.072254 0.037572 0.026012 0.112717 0.066474 0.176301 0.043353 0.052023 0.005780 0.017341 0.017341 0.095376 0.054913 0.078035 0.005780 0.014451
    28 0.040346 0.020173 0.020173 0.040346 0.046110 0.207493 0.023055 0.031700 0.095101 0.069164 0.048991 0.054755 0.063401 0.037464 0.028818 0.043228 0.028818 0.054755 0.005764 0.040346
    29 0.060870 0.002899 0.069565 0.084058 0.052174 0.086957 0.026087 0.049275 0.110145 0.055072 0.017391 0.057971 0.049275 0.063768 0.040580 0.060870 0.020290 0.052174 0.011594 0.028986
    END



## search_matches.E1000.tab
This file contains search hits for all profiles in *extracted_matrix* file with a very lax threshold of E-value<1000.
It means that you should probably filter it by a more stringent E-value.

The sequences of matches can be extracted from all_CDD-3.11.fasta.gz using the start position #. Note that the fasta file contains XXXXX prefix and suffix XXXXX to allow for overhanging profile-sequence alignments.

Columns:

    * Profile ID
    * Score of the hit
    * Evalue of the hit
    * start position # of the hit in the CDD PSSM
    * CDD header

    87	-2.93	814.00	0	gnl|CDD|238014 cd00057, gi,11356716
    87	-2.96	877.00	89	gnl|CDD|238020 cd00063, gi,2497559
    87	-2.64	400.00	0	gnl|CDD|238045 cd00093, gi,7480650
    87	-2.75	536.00	2	gnl|CDD|238050 cd00102, gi,399220,sp,P31746,CDGT_BACS2
    87	-2.80	585.00	0	gnl|CDD|99904 cd00133, gi,20806708,ref,NP_621879.1
    42	-3.35	493.00	0	gnl|CDD|238025 cd00070, gi,3402185,pdb,1A3K
    42	-3.16	338.00	0	gnl|CDD|238025 cd00070, gi,19568746,gb,AAL91920.1,AF479564_1
    42	-3.23	382.00	91	gnl|CDD|238030 cd00075, gi,1666498
    42	-3.36	522.00	71	gnl|CDD|238042 cd00090, gi,15614432,ref,NP_242735.1
    87	-2.69	439.00	0	gnl|CDD|238092 cd00161, gi,17826962,dbj,BAB79291.1
